modules/lib/src/main/resources/assets/js/app/browse/action/MoveContentAction.ts

```diff
@@ -4,27 +4,35 @@ import {ContentSummaryAndCompareStatus} from '../../content/ContentSummaryAndCom
 import {i18n} from 'lib-admin-ui/util/Messages';
 import {ContentTreeGridAction} from './ContentTreeGridAction';
 import {ContentTreeGridItemsState} from './ContentTreeGridItemsState';
+import {ArrayHelper} from 'lib-admin-ui/util/ArrayHelper';
 
-export class MoveContentAction extends ContentTreeGridAction {
+export class MoveContentAction
+    extends ContentTreeGridAction {
 
     constructor(grid: ContentTreeGrid) {
         super(grid, i18n('action.moveMore'));
         this.setEnabled(false);
     }
 
     protected handleExecuted() {
-        const contents: ContentSummaryAndCompareStatus[]
-            = this.grid.getSelectedDataList();
+        const contents: ContentSummaryAndCompareStatus[] = this.grid.getSelectedDataList();
         new MoveContentEvent(contents, this.grid).fire();
     }
 
     isToBeEnabled(state: ContentTreeGridItemsState): boolean {
-        return !state.isEmpty() && !state.isManagedActionExecuting() &&
-               (this.getCurrentSelectedOrHighlightedTotal() < this.grid.getDefaultFullTotal()) &&
-               state.canDelete();
+        return !state.isEmpty() && !state.isManagedActionExecuting() && this.isAnyRootItemNotSelected() && state.canDelete();
     }
 
-    private getCurrentSelectedOrHighlightedTotal(): number {
-        return this.grid.hasHighlightedNode() ? 1 : this.grid.getTotalSelected();
+    private isAnyRootItemNotSelected(): boolean {
+        // if there's at least one non-selected root item then there's at least one option where to move selected items
+        const selectedIds: string[] = this.getHighlightedOrSelectedIds();
+        const rootItemsIds: string[] = this.grid.getRootItemsIds();
+        const diff: string[] = ArrayHelper.difference(rootItemsIds, selectedIds, (a: string, b: string) => a === b);
+
+        return diff.length > 0;
+    }
+
+    private getHighlightedOrSelectedIds(): string[] {
```

Comment:
Any special reason in a different order of words here than in the grid class' method? `getHighlightedOrSelectedIds` vs `getSelectedOrHighlightedItems`.

Reply:
No, will update method's name